option optimize_for = LITE_RUNTIME;

message EatCucumberAnswer
{
 required uint32 result = 1;
 required uint32 power = 2;
 required uint32 yuanbao = 3;
}
